apps/pano/app/routes/send.tsx

```diff
@@ -81,6 +83,13 @@ const Send = () => {
     }
   };
 
+  const [searchParams] = useSearchParams();
+  const queryUrl = searchParams.get("url");
+
+  useEffect(() => {
+    if (queryUrl) fetchMetaTags(queryUrl);
```

Comment:
let's move this to `loader` and return it from there, that way we can ensure that the values are actually generated on the server. then component can use the loader data as inputs' default values.